components/AppShell.vue

```diff
@@ -6,128 +6,64 @@
     >
       <Dialog
         as="div"
-        class="relative z-40 md:hidden"
-        @close="sidebarOpen = false"
+        class="relative md:hidden"
       >
-        <TransitionChild
-          as="template"
-          enter="transition-opacity ease-linear duration-300"
-          enter-from="opacity-0"
-          enter-to="opacity-100"
-          leave="transition-opacity ease-linear duration-300"
-          leave-from="opacity-100"
-          leave-to="opacity-0"
-        >
-          <div class="fixed inset-0 bg-gray-600 bg-opacity-75" />
-        </TransitionChild>
-
-        <div class="fixed inset-0 z-40 flex">
+        <div class="fixed inset-x-0 inset-y-16 flex min-h-full">
           <TransitionChild
             as="template"
             enter="transition ease-in-out duration-300 transform"
-            enter-from="-translate-x-full"
-            enter-to="translate-x-0"
+            enter-from="translate-y-full"
+            enter-to="translate-y-0"
             leave="transition ease-in-out duration-300 transform"
-            leave-from="translate-x-0"
-            leave-to="-translate-x-full"
+            leave-from="translate-y-0"
+            leave-to="translate-y-full"
           >
-            <DialogPanel class="relative flex w-full max-w-xs flex-1 flex-col bg-surface-primary pt-5 pb-4">
-              <TransitionChild
-                as="template"
-                enter="ease-in-out duration-300"
-                enter-from="opacity-0"
-                enter-to="opacity-100"
-                leave="ease-in-out duration-300"
-                leave-from="opacity-100"
-                leave-to="opacity-0"
-              >
-                <div class="absolute top-0 right-0 -mr-12 pt-2">
-                  <!-- FIXME: This should be a button -->
-                  <div
-                    type="button"
-                    class="ml-1 flex h-10 w-10 items-center justify-center rounded-full focus:outline-none focus:ring-2 focus:ring-inset focus:ring-white"
-                    @click="sidebarOpen = false"
-                  >
-                    <span class="sr-only">Close sidebar</span>
-                    <close-outline class="text-content-on-color" />
-                  </div>
-                </div>
-              </TransitionChild>
+            <DialogPanel class="relative flex w-full flex-1 flex-col bg-surface-primary pt-5 pb-4">
               <div class="h-0 flex-1 overflow-y-auto">
                 <nav class="space-y-1 px-2">
-                  <div class="pt-1 pb-4">
-                    <SiteNavigation @link-clicked="sidebarOpen = false" />
-                    <NuxtLink
-                      v-for="item in primaryNavigation"
```

Comment:
`primaryNavigation` has been removed to reduce complexity until we need to extend it. Now "Docs" is just hard coded